Makefile

```diff
@@ -36,10 +35,7 @@ docker-compile:
 # For cases where we're building from local
 docker-build:
 	docker build --rm -t ${IMAGE} rootfs
-
-# Push to a registry that Kubernetes can access.
-docker-push:
-	docker push ${IMAGE}
+	docker tag -f ${IMAGE} ${MUTABLE_IMAGE}
```

Comment:
The `-f` flag is deprecated and produces a warning in current versions of Docker. Maybe we should remove it--AFAICT the behavior is the same either way.